blocks/editable/format-toolbar/index.js

```diff
@@ -66,21 +66,16 @@ class FormatToolbar extends Component {
 		this.setLinkTarget = this.setLinkTarget.bind( this );
 	}
 
-	componentDidMount() {
-		document.addEventListener( 'keydown', this.onKeyDown );
-	}
-
-	componentWillUnmount() {
-		document.removeEventListener( 'keydown', this.onKeyDown );
-	}
-
 	onKeyDown( event ) {
 		if ( event.keyCode === ESCAPE ) {
 			if ( this.state.isEditingLink ) {
 				event.stopPropagation();
 				this.dropLink();
 			}
 		}
+		if ( event.keyCode >= LEFT && event.keyCode <= DOWN ) {
```

Comment:
Are you OK with this? It requires knowledge about the key codes. I'd probably prefer to do four individual or conditions, but I wasn't sure what you'd prefer.

Reply:
I think I'd prefer white listing the keys instead of assuming they are ordered `[ LEFT, DOWN, RIGHT, UP ].indexOf` but it's minor

Reply:
I second making the entire key set explicit.